src/tools/packager.ts

```diff
@@ -38,17 +39,29 @@ function pnpmAvailable() {
   return isPnpm
 }
 
-function detectPackager(): "npm" | "yarn" | "pnpm" {
-  // Expo doesn't support pnpm, so we'll use yarn or npm
-  if (pnpmAvailable()) {
-    return "pnpm"
-  } else if (yarnAvailable()) {
+function detectPackager(): PackagerName {
+  if (yarnAvailable()) {
     return "yarn"
+  } else if (pnpmAvailable()) {
+    return "pnpm"
```

Comment:
[See this comment](https://github.com/infinitered/ignite/pull/1999/files#r950597888) for justification of preferring yarn over pnpm: